client/actor.go

```diff
@@ -66,7 +66,7 @@ func (c *GRPCClient) InvokeActor(ctx context.Context, in *InvokeActorRequest) (o
 		Data:      in.Data,
 	}
 
-	resp, err := c.protoClient.InvokeActor(c.withAuthToken(ctx), req)
+	resp, err := c.protoClient.InvokeActor(ctx, req)
```

Comment:
Not sure why I'm seeing these unrelated changes in this PR?

Reply:
I had to pull in main and fix conflicts from #502. This commit history is messy as a result of this conflict resolution but it does not differ from main.